src/__tests__/parser.ts

```diff
@@ -612,7 +626,7 @@ describe('parser', () => {
       'FunctionalComponentAsConstAsNamedExport',
```

Comment:
maybe it would make sense to rename this test and file it use as it doesn't use the file name anymore.